dali/python/backend_impl.cc

```diff
@@ -443,6 +443,25 @@ void ExposeTensor(py::module &m) {
       layout : str
             Layout of the data
       )code")
+    .def(
+      "_expose_dlpack_capsule",
```

Comment:
I wonder if we need at least a basic test for that in test_backend_impl.py and test_backend_impl_gpu.py.

Reply:
Done